editpost.php

```diff
@@ -43,7 +43,7 @@
 
 if(!$post || ($post['visible'] == -1 && $mybb->input['action'] != "restorepost"))
 {
-	error($lang->error_invalidpost);
+	error(error_message: $lang->error_invalidpost, status_code: 404);
```

Comment:
The first named parameter can be skipped. I have no preference on this matter, so this is more a comment to bring up and decide on a pattern for future code so we follow the same.

@dvz